lib/types/db.ts

```diff
@@ -17,6 +17,7 @@ export type SequelizeAttributes<T extends { [key: string]: any }> = {
 * "xInstance" is the type definition of a fetched record as a Sequelize row instance, which contains some util properties.
 */
 
+// Currencys
```

Comment:
Can these comments just be in singular form? That would match the model name/type names.

Reply:
I changed the comments to `/* */` it just seems more aesthetic to me.